lib/bolt/executor.rb

```diff
@@ -302,11 +302,11 @@ def wait_until_available(targets,
         batch_execute(targets) do |transport, batch|
           with_node_logging('Waiting until available', batch) do
             wait_until(wait_time, retry_interval) { transport.batch_connected?(batch) }
-            batch.map { |target| Result.new(target) }
+            batch.map { |target| Result.new(target, action: 'wait_util_available', object: description) }
           rescue TimeoutError => e
             available, unavailable = batch.partition { |target| transport.batch_connected?([target]) }
             (
-              available.map { |target| Result.new(target) } +
+              available.map { |target| Result.new(action: 'wait_util_available', object: description) } +
```

Comment:
I had a test for this in https://github.com/puppetlabs/bolt/pull/1719/commits/ca6de28c7252dc9bd29fbbe5ac29dc143c25787c

Reply:
Seems like your fix is more extensive than mine, want to pull my change in for the remote transport into taht as well? Or do you want me to just fix the remote stuff and you fix the `wait_until_available` ones?

Reply:
I'm not even sure if @puppetlabs/bolt team has had a chance to look at this. I would defer to them as I'm actually working on different team at Puppet now. This was just a bug I hit when working on a different thing.